### PR TITLE
GoogleAds Connector multiple customer_ids, fix

### LINKED ISSUE #2903: Google Ads Connector says it supports multiple customer ids but only accepts one.
## Expected Behavior
* Google Ads allows taking in multiple customer ids at once.
* Google Ads connector allows taking multiple customer ids at once

## Current Behavior
* Google Ads connector _says_ it allows multiple customer ids at once
* Google Ads connector does not allow taking multiple customer ids at once

## Steps to Reproduce
Screenshot 2021-04-15 at 15 50 13

## Airbyte Version
0.19.0-alpha

## Connector Version (if applicable)
Adwords: 0.2.1

### PR DESCRIPTION
**Pull request by po3na4skld**
## What
Fixing this issue: https://github.com/airbytehq/airbyte/issues/2903

## How
New pattern in spec.json was added in customers_ids field

## Pre-merge Checklist
- [ ] *Run integration tests*
- [ ] *Publish Docker images*

## Recommended reading order
1. spec.json

### CODE REVIEW

File: airbyte-integrations/connectors/source-google-adwords-singer/source_google_adwords_singer/spec.json [hunk 1]
```diff
@@ -47,7 +47,7 @@
       },
       "customer_ids": {
         "type": "string",
-        "pattern": "^[0-9]{10}$",
+        "pattern": "^[0-9]{10}(,[0-9]{10})*$",
```
Comment (michel-tricot) [inline airbyte-integrations/connectors/source-google-adwords-singer/source_google_adwords_singer/spec.json:50]: Can we use actual lists instead? The UI now supports it

**Comment by keu** [comment]
@po3na4skld you should run `./gradlew --no-daemon format --scan` and commit all changed files.

**Comment by po3na4skld** [comment]
> @po3na4skld you should run `./gradlew --no-daemon format --scan` and commit all changed files.

What is the purpose of this? Just wondering

**Comment by po3na4skld** [comment]
> @po3na4skld you should run `./gradlew --no-daemon format --scan` and commit all changed files.
I'm getting this after command had run:
nothing to commit, working tree clean

**Comment by keu** [comment]
the pipeline is failing, https://github.com/airbytehq/airbyte/pull/3154/checks?check_run_id=2517126634
this command is what pipeline run to format your code

**Review by keu — APPROVED** [review]

**Comment by keu** [comment]
@po3na4skld  please bump version and publish connector

**Review by keu — APPROVED** [review]

**Review by yevhenii-ldv — APPROVED** [review]

**Pull request merged**